Backend/app/provider-platform/dynamic-offer-driver-app/Main/src/Domain/Action/UI/Ride/EndRide.hs

```diff
@@ -444,6 +445,14 @@ endRide handle@ServiceHandle {..} rideId req = withLogTag ("rideId-" <> rideId.g
             _ -> throwError $ InternalError (Text.pack $ displayException someException)
         Right resp -> return resp
 
+checkSplLocation :: (MonadThrow m, Log m, MonadTime m, MonadGuid m, EsqDBFlow m r, CacheFlow m r) => Maybe Text -> Text -> m Bool
```

Comment:
@khuzema786 should we consider relocating this function to a shared utility module for improved visibility. @spravinkumar9952